src/tests/end2end/PushConstantTests.cpp

```diff
@@ -49,9 +49,13 @@ class PushConstantTest: public NXTTest {
                 .SetBindingsType(kAllStages, nxt::BindingType::StorageBuffer, 0, extraBuffer ? 2 : 1)
                 .GetResult();
 
-            nxt::PipelineLayout pl = device.CreatePipelineLayoutBuilder()
-                .SetBindGroupLayout(0, bgl)
-                .GetResult();
+            nxt::PipelineLayout pl;
```

Comment:
I think this can use MakeBasicPipelineLayout too.

Reply:
oops, missed this one when factoring it out, thanks.